README.md

```diff
@@ -1,34 +1,18 @@
 # linter-htmlhint
 
-This linter plugin for [Linter](https://github.com/AtomLinter/Linter) provides
-an interface to [htmlhint](https://github.com/yaniswang/HTMLHint). It will be
+A plugin for [Atom Linter](https://github.com/AtomLinter/Linter) providing an interface to [HTMLHint](https://github.com/yaniswang/HTMLHint). It will be
 used with files that have the syntax.
 
 ## Installation
 
-Linter package must be installed in order to use this plugin. If Linter is not
-installed, please follow the instructions [here](https://github.com/AtomLinter/Linter).
-
-### Plugin installation
-
-```ShellSession
+```bash
 $ apm install linter-htmlhint
 ```
 
-### .htmlhintrc
+## Config
 
-This plugin will search for a htmlhint configuration file called `.htmlhintrc`
-and use that file if it exists anywhere in the directory tree. It will stop at
-the first `.htmlhintrc` file found.
+This plugin will search for a [HTMLHint](https://github.com/yaniswang/HTMLHint) configuration file called `.htmlhintrc` and use that file if it exists anywhere in the directory tree. It will stop at the first `.htmlhintrc` file found.
 
 ## Settings
 
-You can configure linter-htmlhint by editing `~/.atom/config.cson` (choose Open
-Your Config in Atom menu):
-
-```coffeescript
-'linter-htmlhint':
-  'htmlhintExecutablePath': null #htmlhint path. run 'which htmlhint' to find the path
-```
-
-[linter]: https://github.com/AtomLinter/Linter "Linter"
```

Comment:
If you leave this line in, and correct the URL as mentioned above you can simply replace all the Linter URLs with `[linter]`. So, for example, your link above would become `[Atom Linter][linter]`. This way you only need to specify the URL once when you are utilizing it multiple times.

(I'm not sure why the current readme specified this, but didn't use it...)